src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -242,6 +241,16 @@ private void updateIdentity() {
             return;
         }
 
+        identity.attach(new IdentityObserver() {
+            @Override
+            public void update() {
+                Platform.runLater(() -> {
+                    alias.setText(identity.getAlias());
+                    updateAvatar(identity);
+                });
+            }
+        });
```

Comment:
With [lambdas](http://www.oracle.com/webfolder/technetwork/tutorials/obe/java/Lambda-QuickStart/index.html) you get a one liner:
```JAVA
identity.attach(() -> Platform.runLater(() -> alias.setText(identity.getAlias)));
```
Your IDE can do that for you.